### DISCUSSION TITLE
Branding for Screenshot tool

### DISCUSSION THREAD

**Discussion by iamhectorsosa** [ekqt/screenshot#64] [General]
Tagging collaborators so far: @kodbilenadam @alefDev-prog @mateusabelli

I'm pretty happy on how the project has been progressing. The purpose of this project is to have a nice tool to create screenshots but most importantly to get a good experience with OSS and to enable others to jump into OSS or further extend their experience with such. I want to thank the three of you for helping out so far. Here are a couple of things I want to share:

* I am changing my username to @iamhectorsosa . As you guys might know, my name is Hector Sosa and it is a bit more explicit. Had issues with other orgs I belong where people just didn't knew it was my based on my previous handle.
* This of course, makes the branding of Screenshot a bit strange. However, I am planning on purchasing a domain for the project. We will redirect the GitHub address using the domain i.e. screenshottool.com/github will go to GitHub repo.

I wanted to ask, what domain suggestions do you have? Here are a couple of options around these brands:
* The Screenshot Tool
* Screenshot Tool
* Screenshot
* Use Screenshot (playful like the React hooks `useScreenshot`)

The domains available are the following (please consider price):
* thescreenshottool.com is available for 20USD
* screenshottool.io is available for 70USD
* screenshottool.app is available for 20USD
* screenshot.style is available for 50USD
* usescreenshot.com is available for 20USD
* usescreenshot.io is available for 70USD
* usescreenshot.app is available for 20 USD (I kinda dig this one)

Wdyt?

On another note guys, we need to start promoting the app as much as we can on socials (I am promoting on Dev.to to get more people to star the repo which will increase visibility and bring more collaborators and users in). Any thoughts on this? It is very organic to suggest the app, because people post screenshots in their blog posts all the time.

**Comment by alefDev-prog** [comment]
Hi, 
Really like the last domain, especially because of its reference to React.  I think Dev.to, and other blog-forums, are really well-suited for the promotion of this particular app. I will surely try to inform people about this app as well!

**Comment by mateusabelli** [comment]
Hello there! thank you for the mention.
I'm learning a lot while contributing with this project!

About the domain name, I feel like the useScreenshot.app is a very nice choice.

**Comment by iamhectorsosa** [comment]
Any thoughts @kodbilenadam

**Reply by kodbilenadam** [reply]
Sorry I missed the notification. useScreenshot.app would be my choice too!

**Reply by kodbilenadam** [reply]
Also the username change is on point imo. It will help branding of the tool as well.

**Reply by iamhectorsosa** [reply]
All good, just wanted to include you in the convo here. Glad you caught up though and that we're all on the same page.

**Comment by iamhectorsosa** [comment]
Alrighty, folks. We bought a domain:
- https://usescreenshot.app
- https://usescreenshot.app/github will direct to the GitHub repository

Please let me know if there are any suggestions or comments about this. Thanks for being part of the early stages of development.
